BUILD.md

```diff
@@ -10,9 +10,11 @@ For macOS, download it at https://hub.docker.com/editions/community/docker-ce-de
 # Clone repo and submodules
 git clone git@github.com:ExodusMovement/mymonero-core-js.git --recursive
 cd mymonero-core-js
+sed -i 's/emconfigure/emmake/' bin/build-emcpp.sh
```

Comment:
Why can't this just be applied to the `bin/build-emcpp.sh` instead of mentioning it in the readme?

Reply:
This command also works in upstream repo.

Reply:
`emmake` works for me

```
emconfigure cmake .. && 
emmake cmake --build . && 
emmake make .
```

Reply:
Does not needed in 1.38.48